### PR TITLE
Support latest version of rest-client

### LINKED ISSUE #16: Support latest version of rest-client
The latest version is 2.0.0

### PR DESCRIPTION
**Pull request by apuntovanini**
This commit fixes #16, tested in my environment and seems to be working fine

### CODE REVIEW

**Comment by apuntovanini** [comment]
@envoy any plans to merge it?

File: eventbrite.gemspec [hunk 1]
```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "rest-client", "~> 1.4"
+  spec.add_dependency "rest-client", ">= 1.4"
```
Comment (kamal) [inline eventbrite.gemspec]: I would prefer this change to be `~> 2.0`

**Review by kamal — CHANGES_REQUESTED** [review]

**Comment by kamal** [comment]
@apuntovanini thanks for the PR! Just left one small comment.

**Comment by apuntovanini** [comment]
@kamal here it is, you're definitely right. Updated PR, guess it's ready to be merged. Will you release a new version with the latest changes? Thanks for maintaining this!

**Comment by kamal** [comment]
@apuntovanini thanks! i'll cut a release in a couple hours

**Comment by apuntovanini** [comment]
@kamal I think you forgot to cut the release! :)

**Comment by kamal** [comment]
@apuntovanini i finally got ownership of the repo and pushed the latest release. Sorry for the delay!

**Pull request merged**